src/app/api/autofill/route.ts

```diff
@@ -10,7 +10,7 @@ export async function GET() {
   }
 
   const res = await fetch(
-    'https://api-beta.copilot.com/v1/custom-fields',
+    `${process.env.COPILOT_API_URL}/v1/custom-fields`,
```

Comment:
lets create this in config file, we seem to have taken this in other places as well. 

config/app.ts
`export const  copilotAPIUrl: process.env.COPILOT_API_URL`